api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -129,7 +129,11 @@ func (in *Operand) DeepCopyInto(out *Operand) {
 			(*out)[key] = val
 		}
 	}
-	in.Spec.DeepCopyInto(&out.Spec)
+	if in.Spec != nil {
+		in, out := &in.Spec, &out.Spec
+		*out = new(runtime.RawExtension)
+		(*in).DeepCopyInto(*out)
+	}
```

Comment:
May I ask that could we use func (in *RawExtension) DeepCopy() *RawExtension{} directly?

https://github.com/kubernetes/apimachinery/blob/15c5dba13c59f42b25aba4e3a6b17061af770216/pkg/runtime/zz_generated.deepcopy.go#L38

Reply:
This file is generated by `controller-gen`, which can't be overwritten by developers.

Reply:
Oh! I see.